From 554c102d3ae616d4ee343874904b1e35c75ed5cd Mon Sep 17 00:00:00 2001
From: Jon Nettleton <jon@solid-run.com>
Date: Tue, 30 May 2023 15:02:38 +0200
Subject: [PATCH] imx8mp-solidrun: For yocto only pass the fdtfile

We should not be setting a subdirectory for the BSP's, this is
a discrepancy with mainline distributions.

Signed-off-by: Jon Nettleton <jon@solid-run.com>
---
 board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c b/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c
index 8ab571a143c..0c2fa3312ea 100644
--- a/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c
+++ b/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c
@@ -642,7 +642,7 @@ int board_late_init(void)
 
 	// auto-select device-tree
 	if (!env_get("fdtfile")) {
-		if(snprintf(fdtfile, sizeof(fdtfile), "freescale/%s-%s.dtb", board_id.som_name, board_id.carrier_name) >= sizeof(fdtfile)) {
+		if(snprintf(fdtfile, sizeof(fdtfile), "%s-%s.dtb", board_id.som_name, board_id.carrier_name) >= sizeof(fdtfile)) {
 			pr_err("%s: buffer too small, fdtfile truncated!\n", __func__);
 		}
 		env_set("fdtfile", fdtfile);
